src/constants/ActionConstants.js

```diff
@@ -51,6 +51,6 @@ export const ACCOUNT_INIT_COMPLETE = 'accountInitComplete'
 
 // OTP
 export const DISABLE_OTP_RESET = 'disableOTPReset'
-
+export const ENABLE_OTP_RESET = 'enableOTPReset'
```

Comment:
This will probably conflict with @wolverineks new action types as string literals.